website/docs/r/organization_token.html.markdown

```diff
@@ -28,6 +28,7 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) A date and time in which the organization token will expire.
```

Comment:
I think that this would be a good place to include information on what format we expect this value to be passed in, maybe with examples. For example, we should specify that this value should be UTC https://hashicorp.atlassian.net/browse/TF-5300

I think it would also be worth mentioning that if this is not set that the token will never expire